mathesar/api/db/viewsets/schemas.py

```diff
@@ -22,7 +22,7 @@ class SchemaViewSet(AccessViewSetMixin, viewsets.GenericViewSet, ListModelMixin,
     access_policy = SchemaAccessPolicy
 
     def get_queryset(self):
-        qs = Schema.objects.all().order_by('-created_at')
+        qs = Schema.objects.all().order_by('name')
```

Comment:
Please undo this change, it seems like it's from the previous issue and doesn't work.